src/GenFu/GenFu.cs

```diff
@@ -151,6 +151,8 @@ public class Defaults
             public static DateTime MIN_DATETIME = DateTime.Now.AddYears(-30);
             public static DateTime MAX_DATETIME = DateTime.Now.AddYears(30);
             
+            public static int MIN_SEED = 2;
```

Comment:
The seed is designed to allow the caller/consumer to determine how many items within the collection will be Null. for example : Min=2, Max: 10 will make (on average) 20% of the list null. This is handy where you might want to simulate a system where you know that only 5% of data has null values; or 95% of data has null values. i have updated by Repos to include a test that shows how this works.